test/unit/polymer.properties-mixin-with-property-accessors.html

```diff
@@ -0,0 +1,357 @@
+
+      class SubElement extends window.MyElement {
+
+        static get properties() {
+          return {
+            prop2: String
+          };
+        }
+
+        constructor() {
+          super();
+          this._calledConstructor++;
+          this.prop = 'sub';
+          this.prop2 = 'prop2';
+        }
+
+        connectedCallback() {
+          super.connectedCallback();
+          this._calledConnectedCallback++;
+        }
+
+        ready() {
+          super.ready();
+          this._calledReady++;
+          this._ensureAttribute('role', 'button');
+          this.addEventListener('click', (e) => this._tapListener(e));
+        }
+
+      }
+
+      customElements.define('sub-element', SubElement);
+
+      window.SubElement = SubElement;
+    });
+    </script>
+  </dom-module>
+
+  <dom-module id="sub-mixin-element">
+    <script>
+    HTMLImports.whenReady(function() {
+
+    function MyMixin(Base) {
+      return class extends Base {
+
+        static get properties() {
+          return {
+            mixin: Number
+          };
+        }
+
+        constructor() {
+          super();
+          this._calledConstructor++;
+          this.mixin = 'mixin';
+        }
+
+        connectedCallback() {
+          super.connectedCallback();
+          this._calledConnectedCallback++;
+        }
+
+        ready() {
+          super.ready();
+          this._calledReady++;
+          this._ensureAttribute('mixinattr', 'mixinattr');
+          this.addEventListener('click', (e) => this._tapListener(e));
+        }
+
+      };
+    }
+
+    class SubMixinElement extends MyMixin(window.SubElement) {
+
+      static get properties() {
+        return {
+          prop3: String,
+          camelCase: String
+        };
+      }
+
+      constructor() {
+        super();
+        this._calledConstructor++;
+        this.prop3 = 'prop3';
+      }
+
+      connectedCallback() {
+        super.connectedCallback();
+        this._calledConnectedCallback++;
+      }
+
+      ready() {
+        super.ready();
+        this._calledReady++;
+        this._ensureAttribute('submixinattr', 'submixinattr');
+        this.addEventListener('click', (e) => this._tapListener(e));
+      }
+
+    }
+
+    customElements.define('sub-mixin-element', SubMixinElement);
+
+    window.SubMixinElement = SubMixinElement;
+
+    });
+    </script>
+  </dom-module>
+
+  <test-fixture id="my-element-attr">
+    <template>
+      <my-element prop="attr"></my-element>
+    </template>
+  </test-fixture>
+
+  <test-fixture id="sub-element-attr">
+    <template>
+      <sub-element prop="attr" prop2="attr"></sub-element>
+    </template>
+  </test-fixture>
+
+  <test-fixture id="sub-mixin-element-attr">
+    <template>
+      <sub-mixin-element prop="attr" prop2="attr" prop3="attr" mixin="5" camel-case="camelCase"></sub-mixin-element>
+    </template>
+  </test-fixture>
+
+  <script>
+  suite('class extends Polymer.PropertiesMixin', function() {
+
+    var el;
```

Comment:
This testsuite still uses vars. Since you updated the vars to consts in the previous file, do it here as well :)

Reply:
Yes!